package.json

```diff
@@ -5,7 +5,7 @@
   "main": "dist/index.js",
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/Decisiv/Development/styled-components-modifiers.git"
+    "url": "git+https://github.com/Decisiv/styled-components-modifiers.git"
```

Comment:
This URL does not need `git+` prefixing it.  `https://github.com/Decisiv/styled-components-modifiers.git` would be sufficient.

Reply:
I think that was auto-generated by npm, but I'll remove it.